.changeset/plenty-emus-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@api3/airnode-node': patch
```

Comment:
hmm, I was thinking this was a patch since the field already existed and all we did was "patch" in the functionality but yeah the case of minor also exists.

Reply:
I think we should stick to semver which dictates that "minor" releases are for new, non-breaking functionality and "patch" releases are for backwards compatible bug fixes. https://semver.org/#summary

I think I would consider this new functionality rather than a bug fix. cc @Siegrift @amarthadan

Reply:
@andreogle is correct. Patch is only for fixing. If there is a bug or security issue that needs to be fixed that's patch. Adding new functionality or changing some functionality (not fixing the problem but changing correct behavior to a different correct behavior) should be minor.

I would also consider this a new functionality.

Reply:
got it changing it to fit that description